### PR TITLE
New Action and Mantis Fix

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Description: 
1. Added new action "Get Insight Details"  and "Get Asset Risks And Vulnerabilities"
2. 0901921 - Connector "Claroty" Data Ingestion map wrong severity in FortiSOAR
3. 0901870 - Connector Claroty Data Ingestion playbook sets alert field as "Ack SLA" and "Response SLA" as MET

Fix: 
1. Marked "Awaiting Action" for field "Ack SLA" and "Response SLA" in sample playbook "Claroty > Ingest" step "Create Record".
2. Set Severity as per Claroty indexing.

### CODE REVIEW

File: claroty/info.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "publisher": "Fortinet",
   "cs_approved": true,
   "cs_compatible": true,
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (cs-omkar-kumbhar) [inline claroty/info.json:8]: did you remove the old doc link?

Comment (cs-ankit-gupta) [reply claroty/info.json:8]: not removed for now, but will add new one once doc link is created

File: claroty/info.json [hunk 2]
```diff
@@ -1499,6 +1499,164 @@
...
+              "title": "",
+              "description": ""
+            }
+          ]
+        }
+      ],
+      "parameters": [
+        {
+          "name": "resource_id",
+          "title": "Asset Resource Id",
```
Comment (cs-prerna-joshi) [inline claroty/info.json:1659]:
```suggestion
          "title": "Asset Resource ID",
```

File: claroty/info.json [hunk 3]
```diff
@@ -1499,6 +1499,164 @@
...
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specifiy asset resource id to get risks and vulnerabilities.",
+          "description": ""
```
Comment (cs-prerna-joshi) [inline claroty/info.json]: please add descriptions for all parameters.

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**